cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -137,4 +137,75 @@ var _ = Describe("[tuningcni]", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 	})
+
+	Context("sysctl allowlist update", func() {
+		var originalSysctls = ""
+
+		BeforeEach(func() {
+			cm, err := client.Client.ConfigMaps("openshift-multus").Get(context.TODO(), "cni-sysctl-allowlist", metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			var ok bool
+			originalSysctls, ok = cm.Data["allowlist.conf"]
+			Expect(ok).To(BeTrue())
+		})
+
+		AfterEach(func() {
+			updateAllowlistConfig(originalSysctls)
+		})
+
+		It("should start a pod with custom sysctl only after adding sysctl to allowlist", func() {
+			macvlanNadName := "macvlan-nad1"
+			updatedSysctls := originalSysctls + "\n" + "^net.ipv4.conf.IFNAME.accept_local$"
+
+			podSysctls, err := networks.SysctlConfig(map[string]string{"net.ipv4.conf.IFNAME.accept_local": "1"})
```

Comment:
make this a const and send it please to the evenExist function 

`net.ipv4.conf.IFNAME.accept_local`